### ISSUE TITLE
Sprite Display Broken

### ISSUE THREAD

**Issue by Mogule7** [ljvmiranda921/sprites-as-a-service#122]
The area where I presume the sprites would be displayed shows a broken image warning.

![image](https://github.com/ljvmiranda921/sprites-as-a-service/assets/120289384/80ef4606-8fb0-4714-add1-93592fbe4bba)

Persists after any interaction with the page.

Have tried on Vivaldi and Edge, including in incognito mode to the same result.

**Comment by Derjyn** [comment]
Yup, it's busted. Inspecting it, seems the Base64 string isn't being output correctly:

```
<img data-v-1a93f51a="" src="data:image/png;base64," class="display">
```

There are also numerous errors which seem to be CORS related. For example:

```
error.response 
  GET https://sprites-as-a-service-tblytwilzq-ue.a.run.app/api/v1/sprite?size=300
net::ERR_FAILED 404 (Not Found)
```

<img width="728" height="322" alt="Image" src="https://github.com/user-attachments/assets/a158682b-5ad8-498c-adae-e1a9bad80ac5" />
